website/docs/cloud-docs/agents/telemetry.mdx

```diff
@@ -4,40 +4,104 @@ page_title: Telemetry - Terraform Cloud Agents - Terraform Cloud and Terraform E
 
 # Telemetry
 
-Terraform Cloud Agents emit telemetry data using the [OpenTelemetry](https://opentelemetry.io/) protocol.
+Terraform Cloud Agents may be configured to emit helpful telemetry data,
+including both trace spans and metrics.
```

Comment:
the instrumentation technically has the ability to emit logging as well, and is enabled/configured to be possible within the TFC agent currently, right? I can't recall if i've verified this prior.

Reply:
We do not use OpenTelemetry's logging facilities at all, primarily because even at the protocol level they are still [marked as experimental](https://opentelemetry.io/docs/reference/specification/logs/).